persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -77,16 +74,16 @@ public static void assertGeneratedSources(String subDir, Command cmd) {
             Assert.assertTrue(Files.exists(actualOutputFile));
             Assert.assertEquals(readContent(actualOutputFile), readContent(expectedOutputFile));
         }
-        if (!(subDir.equals("tool_test_generate_4") || subDir.equals("tool_test_generate_26") ||
-                subDir.equals("tool_test_generate_18"))) {
-
-            BuildProject buildProject = BuildProject.load(Paths.get(GENERATED_SOURCES_DIRECTORY).resolve(subDir)
-                    .toAbsolutePath());
-            Package currentPackage = buildProject.currentPackage();
-            PackageCompilation compilation = currentPackage.getCompilation();
-            Assert.assertFalse(compilation.diagnosticResult().hasErrors());
-        }
+//        if (!(subDir.equals("tool_test_generate_4") || subDir.equals("tool_test_generate_26") ||
+//                subDir.equals("tool_test_generate_18"))) {
+
+//            BuildProject buildProject = BuildProject.load(Paths.get(GENERATED_SOURCES_DIRECTORY).resolve(subDir)
+//                    .toAbsolutePath());
+//            Package currentPackage = buildProject.currentPackage();
+//            PackageCompilation compilation = currentPackage.getCompilation();
+//            Assert.assertFalse(compilation.diagnosticResult().hasErrors());
```

Comment:
Any reason to remove the build after client generation?

Reply:
We couldn't enable this until all the required changes are done from the moudle-persist

Reply:
Enabled now in [e31880d](https://github.com/ballerina-platform/persist-tools/pull/106/commits/e31880d0f4165a2a2c2e1390f97188025e03fbe9)